homeassistant/components/alexa/config.py

```diff
@@ -53,7 +53,7 @@ def is_reporting_states(self):
             )
         try:
             await self._unsub_proactive_report
-        except Exception:  # pylint: disable=broad-except
```

Comment:
Why is this not needed?

Reply:
Because we raise in the exception handler.

Reply:
Oh!